internal/evalhub_mcp/config/config_test.go

```
@@ -0,0 +1,341 @@
+	if cfg.Transport != "stdio" {
+		t.Errorf("expected default transport, got %q", cfg.Transport)
+	}
+}
+
+// helpers
+
+func clearEnv(t *testing.T) {
+	t.Helper()
+	for _, key := range []string{"EVALHUB_BASE_URL", "EVALHUB_TOKEN", "EVALHUB_TENANT", "EVALHUB_INSECURE"} {
+		t.Setenv(key, "")
+		os.Unsetenv(key)
```

golangci-lint (2.11.4)

[error] Error return value of `os.Unsetenv` is not checked

(errcheck)